commons-numbers-fraction/src/test/java/org/apache/commons/numbers/fraction/CommonTestCases.java

```diff
@@ -570,6 +570,7 @@ private CommonTestCases() {}
         testCases.add(new BinaryIntOperatorTestCase(0, 1, Integer.MAX_VALUE, 0, 1));
         testCases.add(new BinaryIntOperatorTestCase(0, -1, Integer.MAX_VALUE, 0, 1));
 
+        testCases.add(new BinaryIntOperatorTestCase(1, 1, Integer.MIN_VALUE, 1, 1));
```

Comment:
I would add all the cases for completeness:
```
1, 1
1, -1
-1, 1
-1, -1
```

These should all evaluate to 1 / 1.

Reply:
@aherbert done.